extensions/timecamp/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "timecamp",
+  "title": "Timecamp",
+  "description": "Start and stop timecamp tasks.",
+  "icon": "timecamp-logo.png",
+  "author": "iAmKevinCruz",
+  "categories": [
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Start/Stop Tasks",
+      "description": "Start/Stop tasks and edit active timer's notes",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "timecamp_api_token",
+      "description": "Get your programming API token from https://app.timecamp.com/app#/settings/users/me",
```

Comment:
Could you add a placeholder to show a mock api key 🙂

Reply:
Taken care of :)